client/src/utils/spotify.js

```diff
@@ -15,15 +15,18 @@ const Spotify = {
       const expiresIn = Number(tokenExpiresInMatch[1]);
       window.setTimeout(() => { accessToken = ''; }, expiresIn * 1000);
       window.history.pushState('Access Token', null, '/');
-      return accessToken.substring(1)
-        .split('&')
-        .reduce(function (initial, item) {
-          if (item) {
-            var parts = item.split('=');
-            initial[parts[0]] = decodeURIComponent(parts[1]);
-          }
-          return initial;
-        }, {});;
+      //this returns an object with the access token
```

Comment:
Okay, I'm not sure why you have this file, because all this stuff with the Spotify Access Tokens and such are all actions we agreed has to happen on the backend.

Reply:
![image](https://user-images.githubusercontent.com/21187304/55429932-a8dd3e00-555a-11e9-8734-f87f505f8aa9.png)
I just wanted to get the code that you get from the auth process, not unless I'm misunderstanding what's happening.